pr_testing/test_multiple_prs.sh

```diff
@@ -786,6 +787,30 @@ if ! $CMSDIST_ONLY ; then # If a CMSSW specific PR was specified #
     exit 0
   fi
 
+  if $PRODUCTION_RELEASE ; then
+    pushd ..
+      mv src src.tmp
+      mkdir src && cd src
+      DSIZE=0
+      THRDS=""
+      NUM_PROC=$(nproc)
+      if [ $NUM_PROC = "0" ] ; then NUM_PROC=1; fi
+      if $USE_IB_TAG ; then git cms-init --upstream-only $CMSSW_IB ; else git cms-init --upstream-only ; fi
+      git repack -h 2>&1 | grep '\-\-threads' && THRDS="--threads ${NUM_PROC}" || true
+      git repack -a -d ${THRDS}
+      git repack -a -d ${THRDS}
+      OSIZE=$(du -sk .git/objects/pack | sed 's|\s.*||')
+      PR_NR=$(echo ${PULL_REQUEST} | sed 's/.*#//' )
```

Comment:
@iarspider , are we sure `${PULL_REQUEST}` is a cmssw PR? This whole section should be run for ` $PRODUCTION_RELEASE` and ` ${PULL_REQUEST}` being cms-sw/cmssw PR